### PR TITLE
Fix python wrapper swig compile error

### PR DESCRIPTION
**Pull request by Presburger**
issue: #209
Signed-off-by: Yusheng.Ma <Yusheng.Ma@zilliz.com>

### CODE REVIEW

**Comment by cydrain** [comment]
There are 2 other things need to be done:
1. add this directory into ".github/workflows/ut.yaml" to track any code change under this directory
2. add swig compile check into CI

File: .github/workflows/ut.yaml [hunk 1]
```diff
@@ -50,6 +51,16 @@ jobs:
             ./output/unittest/faiss_test
           fi
           ./output/unittest/test_knowhere
+      - name: swig compile
+        run: |
+          sudo apt install -y libpcre-dev libpython3-dev
```
Comment (cydrain) [inline .github/workflows/ut.yaml:56]: move these to "scripts/install_deps.sh"

File: .github/workflows/ut.yaml [hunk 2]
```diff
@@ -50,6 +51,16 @@ jobs:
             ./output/unittest/faiss_test
           fi
           ./output/unittest/test_knowhere
+      - name: swig compile
+        if: ${{ matrix.os == 'ubuntu-18.04' }}
+        run: |
+          wget https://downloads.sourceforge.net/swig/swig-4.0.2.tar.gz
```
Comment (cydrain) [inline .github/workflows/ut.yaml:57]: move this to "scripts/install_deps.sh"

File: .github/workflows/ut.yaml [hunk 3]
```diff
@@ -50,6 +51,25 @@ jobs:
             ./output/unittest/faiss_test
           fi
           ./output/unittest/test_knowhere
+
+  swig-build:
+    name: Python3 SWIG compile
+    runs-on: ubuntu-20.04
```
Comment (cydrain) [inline .github/workflows/ut.yaml:57]: use ubuntu-18.04

Comment (Presburger) [reply .github/workflows/ut.yaml:57]: ubuntu-18.04 python version is so slow.

**Comment by cydrain** [comment]
- name: Build test image
        run: |
          ./build.sh -t Release -u
      - name: Build Swig
        run: |
          ./build.sh -t Release -u
      - name: Run Unittest
        run: |
          if [[ -f "output/unittest/faiss_test" ]]; then
            ./output/unittest/faiss_test
          fi
            ./output/unittest/test_knowhere

**Pull request merged**